### ISSUE TITLE
IIGO is broken

### ISSUE THREAD

**Issue by mpardalos** [SOMAS2020/SOMAS2020#126]
https://github.com/SOMAS2020/SOMAS2020/blob/5b8ebfbe193484b5c3389d363b6df936d9db3125/internal/server/iigo.go#L14

This line ignores errors from `iigointernal.runIIGO()`. If you make the error a panic then you'll realise that IIGO is broken. 

This is because `judgePointer`, `speakerPointer`, and `presidentPointer` are all initialised to `nil`:
https://github.com/SOMAS2020/SOMAS2020/blob/5b8ebfbe193484b5c3389d363b6df936d9db3125/internal/server/iigointernal/orchestration.go#L64-L66

Here is the log from that:

```
go run .
2020/12/24 01:28:14 [SERVER]: start runTurn
2020/12/24 01:28:14 [SERVER]: TURN: 1, Season: 1
2020/12/24 01:28:14 [SERVER]: start startOfTurnUpdate
2020/12/24 01:28:14 [Team4]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [Team1]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [Team3]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [Team5]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [Team2]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [Team6]: Received start of turn game state update: gamestate.ClientGameState{Season:0x1, Turn:0x1, ClientInfo:gamestate.ClientInfo{Resources:100, LifeStatus:Alive, CriticalConsecutiveTurnsCounter:0x0}}
2020/12/24 01:28:14 [SERVER]: finish startOfTurnUpdate
2020/12/24 01:28:14 [SERVER]: start runOrgs
2020/12/24 01:28:14 [SERVER]: start runIIGO
2020/12/24 01:28:14 [SERVER]: finish runOrgs
2020/12/24 01:28:14 [SERVER]: finish runTurn
panic: Could not run IIGO since President has no resoruces to spend

goroutine 1 [running]:
github.com/SOMAS2020/SOMAS2020/internal/server.(*SOMASServer).runIIGO(0xc000110000, 0x0, 0x0)
	/home/mpardalos/Documents/go/src/SOMAS2020/internal/server/iigo.go:15 +0x198
github.com/SOMAS2020/SOMAS2020/internal/server.(*SOMASServer).runOrgs(0xc000110000, 0x0, 0x0)
	/home/mpardalos/Documents/go/src/SOMAS2020/internal/server/turn.go:39 +0xd7
github.com/SOMAS2020/SOMAS2020/internal/server.(*SOMASServer).runTurn(0xc000110000, 0x0, 0x0)
	/home/mpardalos/Documents/go/src/SOMAS2020/internal/server/turn.go:22 +0x19f
github.com/SOMAS2020/SOMAS2020/internal/server.(*SOMASServer).EntryPoint(0xc000110000, 0xc000110000, 0x0, 0x0, 0x0, 0x0)
	/home/mpardalos/Documents/go/src/SOMAS2020/internal/server/server.go:63 +0x565
main.main()
	/home/mpardalos/Documents/go/src/SOMAS2020/main.go:62 +0x4a
exit status 2
```

**Comment by lhl2617** [comment]
CC author @preetl @Neelesh99 approver @Eirikalb  

#91. Bug was due to unresolved comments on PR.

**Comment by Neelesh99** [comment]
Fixed in #148
